include/soci/sqlite3/soci-sqlite3.h

```diff
@@ -307,6 +307,21 @@ struct sqlite3_session_backend : details::session_backend
         return "select name as \"TABLE_NAME\""
                 " from sqlite_master where type = 'table'";
     }
+    std::string get_column_description_query()
+    {
+        return "select name as 'COLUMN_NAME',"
```

Comment:
I think this could be a bit more readable using raw C++11 string literals (`R"(...)"`).

Reply:
As you want, I missed the override specifier on the function definition.
For the string, the only change will be the escaped  quote around `notnull`, which is fairly contained at the end of the query.

Reply:
I missed the const too, because it doesn't override at the moment :(